plugins/docker/Dockerfile

```diff
@@ -34,6 +34,7 @@ RUN pip install git+https://github.com/openwallet-foundation/acapy-plugins@1.2.4
 RUN pip install git+https://github.com/openwallet-foundation/acapy-plugins@1.2.4#subdirectory=connection_update
 RUN pip install git+https://github.com/openwallet-foundation/acapy-plugins@1.2.4#subdirectory=multitenant_provider
 RUN pip install git+https://github.com/openwallet-foundation/acapy-plugins@1.2.4#subdirectory=rpc
+RUN pip install git+https://github.com/OpSecId/acapy-plugins@add-did-update#subdirectory=webvh
```

Comment:
Note addition of connections plugin at https://github.com/bcgov/traction/pull/1547/files
`RUN pip install git+https://github.com/openwallet-foundation/acapy-plugins@1.2.4.1#subdirectory=connections`
this is to preserve 1.0 connections ability with the aca-py 1.3.0 release that removes them.

As well, if we're merging an RC, I don't think there's usually a plugin tag until the non-RC release is done, so normally we'd adjust all the plugins to the 1.3.0 tag instead of keeping 1.2.4, so we might need to verify if there's any necessary changes or anything (don't think so with the testing I did). Not sure if the webvh plugin here would be in the upcoming tag as well?

cc @esune

Reply:
This is merged to `acapy-plugins` now, so could be installed from there. After `1.3.0` is released we will create a plugins `1.3.0` release and then this will need to be updated again.

Reply:
1.3.0 tag is in now so this (and other plugin refs) can use that version now
https://github.com/openwallet-foundation/acapy-plugins/releases/tag/1.3.0